tests/sensors/test_base_sensor.py

```diff
@@ -464,3 +464,31 @@ def test_reschedule_with_test_mode(self):
                 self.assertEqual(len(task_reschedules), 0)
             if ti.task_id == DUMMY_OP:
                 self.assertEqual(ti.state, State.NONE)
+
+    def test_sensor_with_negetive_poke_interval(self):
+        negative_poke_interval = -10
+        positive_poke_interval = 10
+        with self.assertRaises(AirflowException):
+            sensor = self._make_sensor(
+                return_value=None,
+                poke_interval=negative_poke_interval,
+                timeout=25)
+        sensor = self._make_sensor(
+            return_value=None,
+            poke_interval=positive_poke_interval,
+            timeout=25)
+        self.assertEqual(sensor.poke_interval, positive_poke_interval)
```

Comment:
I don't see the point of having this `assertEqual` line. The fact that `self._make_sensor()` itself can run without exception is good enough I think.